src/lib/helpers/AxesHelper.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import { AxesHelper } from 'three'
+  import Object3DInstance from '../instances/Object3DInstance.svelte'
+
+  //self
+  export let length:number = 1;
+
+  let helper: AxesHelper = new AxesHelper(length);
+
+</script>
+
+{#if helper}
+  <Object3DInstance object={helper} />
```

Comment:
I'd say since this one is not bound to an object you might want to transform it. Pass down all props of `<Object3DInstance>`.